src/notation/view/notationnavigator.cpp

```diff
@@ -21,14 +21,43 @@
  */
 #include "notationnavigator.h"
 
+#include "draw/types/geometry.h"
 #include "engraving/dom/system.h"
 
 #include "log.h"
+#include <memory>
+#include <qevent.h>
+#include <qpainter.h>
+#include <qquickitem.h>
+#include <qquickpainteditem.h>
+#include <qwidget.h>
 
 using namespace mu::notation;
 
+NotationNavigatorViewRect::NotationNavigatorViewRect(QQuickItem* parent)
+    : QQuickPaintedItem(parent)
+{
+}
+
+void NotationNavigatorViewRect::paint(QPainter* painter)
+{
+    TRACEFUNC;
+
+    QColor color(configuration()->selectionColor());
+    QPen pen(color, configuration()->borderWidth());
+    painter->setPen(pen);
+    painter->setBrush(QColor(color.red(), color.green(), color.blue(), configuration()->cursorOpacity()));
+
+    painter->drawRect(m_cursorRect.toQRectF());
+}
+
+void NotationNavigatorViewRect::setRect(RectF cursorRect)
```

Comment:
We usually pass RectF by const reference rather than by copying:
```suggestion
void NotationNavigatorViewRect::setRect(const RectF& cursorRect)
```